test/e2e/fixtures.go

```diff
@@ -27,6 +27,8 @@ import (
 
 	"k8s.io/component-base/featuregate"
 
```

Comment:
remove this extra empty line. these two k8s.io packages should be in one import section.

Reply:
done.